Source/Core/Core/FifoPlayer/FifoPlayer.cpp

```diff
@@ -189,6 +186,18 @@ bool FifoPlayer::IsRunningWithFakeVideoInterfaceUpdates() const
   return m_File->ShouldGenerateFakeVIUpdates();
 }
 
+u32 FifoPlayer::GetMaxObjectCount() const
+{
+  u32 result = 0;
+  for (u32 i = 0; i < m_FrameInfo.size(); i++)
```

Comment:
Would this not be able to be a ranged for loop? If not, the iteration variable should likely be typed as a `size_t` instead.